src/config.rs

```diff
@@ -8,6 +8,7 @@ pub struct Config {
     pub prefix: PathBuf,
     pub registry_index_path: PathBuf,
     pub registry_url: Option<String>,
+    pub registry_api_host: String,
```

Comment:
theoretically we could also pull this from the local crates.io index config, but in the preparation for splitting web/build/watcher servers, we wouldn't have the index locally available any more.

Reply:
Should we centralize on this for the other accessors too then? The crate data update during a build will have an index to pull the url from, but it might be less confusing if it uses this too.

Reply:
during the build we get the info from the config in the local index, which feels like the more correct data source. 

But since this piece here is running on the webserver, so doesn't have the local index, I added it as a separate setting.